src/ios/FirebasePlugin.h

```diff
@@ -3,6 +3,7 @@
 
 @interface FirebasePlugin : CDVPlugin
 + (FirebasePlugin *) firebasePlugin;
+- (void)getVerificationID:(CDVInvokedUrlCommand*)command;

```

Comment:
I think this method should be named `verifyPhoneNumber` since that is the name used for the equivalent android method: https://github.com/arnesson/cordova-plugin-firebase#verifyphonenumber-android-only

Reply:
I agree. but, you should preserve the method for backward compatibility.
There was 140k downloads since f9b7e80ab69bd840e6e4e638f069c992ed2901cb .

Reply:
fair enough